Makefile

```diff
@@ -3,7 +3,7 @@
 IMG ?= controller:latest
 
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
-ENVTEST_K8S_VERSION = 1.31.1
+ENVTEST_K8S_VERSION = 1.32.0
```

Comment:
Will need envtests bin to be pushed to the bucket. I'm seeing to that.
I plan to push 1.32.1 so I suggest to switch to that here as well.